### ISSUE TITLE
Cannot read properties of undefined (reading 'name')

### ISSUE THREAD

**Issue by nadirnasir** [getinsomnia/insomnia#6277]
### Expected Behavior

Insomnia just got updated and demanded restart. I restart and hit 'send' on a saved api call. App crashed as soon as it receives a response form the api.
TypeError: Cannot read properties of undefined (reading 'name') at hde (file:///C:/Users/***/AppData/Local/insomnia/app-2023.5.0/resources/app.asar/index-13ca93ca.js:84:34818) at async X8 (file:///C:/Users/***/AppData/Local/insomnia/app-2023.5.0/resources/app.asar/index-13ca93ca.js:84:33181) at async Module.Ie (file:///C:/Users/***/AppData/Local/insomnia/app-2023.5.0/resources/app.asar/request-7bcb48dc.js:1:7088) at async Sg (file:///C:/Users/***/AppData/Local/insomnia/app-2023.5.0/resources/app.asar/index-13ca93ca.js:31:3691) at async Ke (file:///C:/Users/***/AppData/Local/insomnia/app-2023.5.0/resources/app.asar/index-13ca93ca.js:30:18467)

![image](https://github.com/Kong/insomnia/assets/909956/1708e05c-4d99-4b71-8944-e8e6a81f3561)

### Actual Behavior

Usually the app shows the API response properly.

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

2023.5.0

### What operating system are you using?

Windows

### Operating System Version

Windows 10

### Installation method

downloaded from insomnia website. The error happened after app updated itself.

### Last Known Working Insomnia version

2023.5.0

**Comment by jackkav** [comment]
@nadirnasir Thanks for the report could you describe anything about the request that would help us to reproduce this issue? Any authentication? headers? templating? chains requests?

**Comment by ikkira** [comment]
I also experience a similar situation. The application only functions properly after I deactivate the "insomnia-plugin-save-variables" plugin.

**Comment by g-industrialist** [comment]
> I also experience a similar situation. The application only functions properly after I deactivate the "insomnia-plugin-save-variables" plugin.

I dont have this plugin "insomnia-plugin-save-variables", but still fail in 2023.5.0. **2023.4.0** worked very well (now I go to downgrade it first).

**Comment by g-industrialist** [comment]
https://github.com/Kong/insomnia/issues/6277#issuecomment-1678263724


```
curl --request GET \
  --url https://petstore.swagger.io/v2/user/string
```
even run petstore sample api , it still fails.

**Comment by nadirnasir** [comment]
> @nadirnasir Thanks for the report could you describe anything about the request that would help us to reproduce this issue? Any authentication? headers? templating? chains requests?

> I also experience a similar situation. The application only functions properly after I deactivate the "insomnia-plugin-save-variables" plugin.

@jackkav  I did not do anything special, just a standard POST request I had been using for past couple of weeks constantly during a development project. The only change was the app update notification. After restart the app failed. The API also didn't send any bad data, it was astandard json.

But I think @ikkira is actually is correct. I disabled the installed plugins and the app started working again. @ikkira thanks for the pointer. 

These are the plugins I had been using.

- insomnia-plugin-reqresp-base64
- insomnia- plugin-timestamp-expander

Had to disabled both one by one to get everything back to normal.
